com.unity.render-pipelines.core/Runtime/PostProcessing/Shaders/LensFlareCommon.hlsl

```diff
@@ -3,17 +3,6 @@
 #include "Packages/com.unity.render-pipelines.core/ShaderLibrary/Random.hlsl"
 #include "Packages/com.unity.render-pipelines.core/ShaderLibrary/Sampling/Sampling.hlsl"
 
-#if SHADER_API_GLES
-struct AttributesLensFlare
-{
-    float4 positionCS       : POSITION;
-    float2 uv               : TEXCOORD0;
-
-#ifndef FLARE_PREVIEW
-    UNITY_VERTEX_INPUT_INSTANCE_ID
-#endif
-};
-#else
```

Comment:
What is the reason to remove this? it seems this was introduced to fix GLES2 compilation error.

Reply:
[Fixed]